integration-tests/README.md

```diff
@@ -26,12 +26,18 @@ weblogic-kubernetes-operator/integration-tests/src/test/resources - properties,
 - Clone the `weblogic-kubernetes-operator` repository.
 - Set up a personal account on `container-registry.oracle.com`.
 - Log in to `https://container-registry.oracle.com` and accept Oracle Standard Terms and Restrictions to pull WebLogic images from `container-registry.oracle.com/middleware/weblogic`.
-- Export the following environment variables before running the tests:
+- Following environment variables must be set before running the tests. BASE_IMAGES_REPO variables are used to download the Oracle public images. TEST_IMAGES_REPO variable are used to store any public image used to certify the WebLogic Kubernates Operator (such as NGNIX Controller, Prometheous etc.) and to store the intermediate WebLogic domain images while running the integration tests.
```

Comment:
```suggestion
- Following environment variables must be set before running the tests. BASE_IMAGES_REPO variables are used to download the Oracle public images. TEST_IMAGES_REPO variables are used to store any public images used to certify the WebLogic Kubernates Operator (such as NGNIX Controller, Prometheous etc.) and to store the intermediate WebLogic domain images while running the integration tests.
```

Reply:
Modified the README.md as per suggestion